src/main/java/fi/metatavu/dcfb/server/locations/LocationConsts.java

```diff
@@ -0,0 +1,12 @@
+package fi.metatavu.dcfb.server.locations;
+
+public class LocationConsts {
+
+  public static final String COORDINATE_REFERENCE_SYSTEM = "epsg4326";
+  public static final String[] SUPPORTED_COORDINATE_REFERENCE_SYSTEMS = {"epsg4326"}; 
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this member "protected". ([squid:S2386](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS2386))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQt_bhzYwZU9vvrs3K3&open=AWQt_bhzYwZU9vvrs3K3)

[ISSUE_KEY]: AWQt_bhzYwZU9vvrs3K3